src/baldr/pathlocation.cc

```diff
@@ -46,7 +46,7 @@ bool PathLocation::shares_edges(const PathLocation& other) const {
     for (const auto& other_edge : other.edges) {
       if (edge.id == other_edge.id && edge.sos == other_edge.sos &&
           midgard::equal<float>(edge.percent_along, other_edge.percent_along) &&
-          midgard::similar<float>(edge.distance + 1, other_edge.distance + 1) &&
+          midgard::equal<float>(edge.distance, other_edge.distance, .1f) &&
```

Comment:
change the distance equality check to be down to 10 centimeters so that floating point noise on 32bit platforms near the equator doesnt trip up distance checks. the 7th decimal place seems to be where this starts cropping up in ll distance computations which at the equator its close to 11 centimeters.